src/main/java/ReviewResponse.java

```diff
@@ -1,10 +1,13 @@
 import lombok.Data;
 import lombok.Getter;
+import lombok.Setter;
 import lombok.ToString;
 
 import java.util.List;
 
 @ToString
+@Setter
+@Getter
 @Data
```

Comment:
:warning: Using Lombok's @Data annotation can lead to violation of the Single Responsibility Principle (SRP) as it generates equals, hashCode, and toString methods automatically. (Severity: Major)